templates/rhpam73-kieserver-externaldb.yaml

```diff
@@ -276,32 +276,32 @@ parameters:
   value: ''
   required: false
 - displayName: Use the secure route name to link application requests to KIE_SERVER_HOSTNAME_HTTPS vs KIE_SERVER_HOSTNAME_HTTP.
-  description: 'If true, will use secure-APPLICATION_NAME-kieserver vs. APPLICATION_NAME-kieserver as the route name.'
+  description: 'If true, the KIE server will report secure-APPLICATION_NAME-kieserver vs. APPLICATION_NAME-kieserver as the route name in its API, including a connection to Business Central or Business Central Monitoring.'
```

Comment:
This is incorrect. It would be better stated like this:

"If true, the KIE server will use secure-APPLICATION_NAME-kieserver vs. APPLICATION_NAME-kieserver to lookup the KIE server Route for Business Central to report.'

We use websockets for connections between BC and KIE server, not REST. So from the user's perspective this change is all about which URL will show up in BC for them to click on when viewing the kie server list.